docs/src/config/core-components.adoc

```diff
@@ -63,212 +64,237 @@ of the low level motion planner.
-    bit is TRUE, the feed rate is set to 0.
+  (bit, in) When Feed Stop Control is enabled with 'M53 P1', and this
+  bit is TRUE, the feed rate is set to 0.
 
 * 'motion.feed-inhibit' -
-     (bit, in) When this bit is TRUE, the feed rate is set to 0.
-    This will be delayed during spindle synch moves till the end of the move.
+  (bit, in) When this bit is TRUE, the feed rate is set to 0.
+  This will be delayed during spindle synch moves till the end of the move.
 
 * 'motion.in-position' -
-    (bit, out) TRUE if the machine is in position.
+  (bit, out) TRUE if the machine is in position.
 
 * 'motion.motion-enabled' -
-    (bit, out) TRUE when in 'machine on' state.
+  (bit, out) TRUE when in 'machine on' state.
 
 * 'motion.motion-type' -
-    (s32, out) These values are from src/emc/nml_intf/motion_types.h
-
- - 0: Idle (no motion)
-
- - 1: Traverse
-
- - 2: Linear feed
-
- - 3: Arc feed
-
- - 4: Tool change
-
- - 5: Probing
-
- - 6: Rotary axis indexing
+  (s32, out) These values are from src/emc/nml_intf/motion_types.h
+  - 0: Idle (no motion)
+  - 1: Traverse
+  - 2: Linear feed
+  - 3: Arc feed
+  - 4: Tool change
+  - 5: Probing
+  - 6: Rotary axis indexing
+
+* 'motion.on-soft-limit' - (bit, out) TRUE when the machine is on a soft limit.
+
+* 'motion.probe-input' - (bit, in)
+  'G38.n' uses the value on this pin to determine when the probe has made contact.
+  TRUE for probe contact closed (touching), FALSE for probe contact open.
+
+* 'motion.program-line' - (s32, out) The current program line while executing.
+  Zero if not running or between lines while single stepping.
+
+* 'motion.requested-vel' - (float, out)
+  The current requested velocity in user units per second.
+  This value is the F-word setting from the G-code file, possibly reduced to accommodate machine velocity and acceleration limits.
+  The value on this pin does not reflect the feed override or any other adjustments.
+
+* 'motion.teleop-mode' - (bit, out) TRUE when motion is in 'teleop mode', as opposed to 'coordinated mode'
+
+* 'motion.tooloffset.x ... motion.tooloffset.w' - (float, out, one per axis) shows the tool offset in effect;
+  it could come from the tool table ('G43' active), or it could come from the gcode ('G43.1' active)
+
+* 'spindle.0.at-speed' - (bit, in)
+  Motion will pause until this pin is TRUE, under the following conditions:
+** before the first feed move after each spindle start or speed change;
+** before the start of every chain of spindle-synchronized moves;
+** and if in CSS mode,
+  at every rapid to feed transition.
+  +
+  This input can be used to ensure that the spindle is up to speed before starting a cut, or that a lathe spindle in CSS mode has
+  slowed down after a large to small facing pass before starting the next pass at the large diameter.
+  Many VFDs have an 'at speed' output.
+  Otherwise, it is easy to generate this signal with the 'HAL near' component, by comparing requested and actual spindle speeds.
```

Comment:
Where do you have this part from?

Reply:
@smoe

Reply:
Cannot recall. Was is possibly just moved somehow within the same file?